src/modules/services/submodule/mainnetSwap/components/SelectInputAccountOutgoingSwap.vue

```diff
@@ -163,7 +164,7 @@ export default defineComponent({
       }
       selectedAccountBalance.value = otherToken.value!='prx.xpx'? 
       accounts.find(account => account.address == accountAddress).assets.find(asset=>asset.namespaceNames==otherToken.value)? 
-      accounts.find(account => account.address == accountAddress).assets.find(asset=>asset.namespaceNames==otherToken.value).amount/Math.pow(10,6) : 0 
+      accounts.find(account => account.address == accountAddress).assets.find(asset=>asset.namespaceNames==otherToken.value).amount/Math.pow(10,divisibility.value) : 0 
```

Comment:
why not use a cleaner code ? just find the asset first, then only proceed to next step. It is better to use asset Id instead of namespace and you are comparing a string array to a string ?